### PR TITLE
CGT-652: Flat Rate Calculation Logic

### PR DESCRIPTION
**Pull request by miranda-hawkes**

### CODE REVIEW

File: app/models/CalculationResultModel.scala [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import play.api.libs.json.Json
+
+case class CalculationResultModel(taxOwed: Double,
```
Comment (jameshforster) [inline app/models/CalculationResultModel.scala]: Needs to be BigDecimals not Doubles

File: conf/app.routes [hunk 2]
```diff
@@ -1,3 +1,4 @@
 # microservice specific routes
 
-GET        /calculate             controllers.CalculatorController.calculate(left: Int ?= 0, right: Int ?= 0)
+GET        /calculate             controllers.CalculatorController.calculate(customerType: String, priorDisposal: String, annualExemptAmount: Option[Double], isVulnerable: Option[String],currentIncome: Double,personalAllowanceAmt: Double,disposalValue: Double,disposalCosts: Double,acquisitionValueAmt: Double,acquisitionCostsAmt: Double,improvementsAmt: Double,reliefs: Double,allowableLossesAmt: Double,entReliefClaimed: String)
```
Comment (jameshforster) [inline conf/app.routes]: Should receive equivalent of SummaryModel found in frontend?

File: app/models/CalculationResultModel.scala (hunk 1 above)
Comment (MJCallahanPage) [inline app/models/CalculationResultModel.scala]: Not changed, based on testing.

File: conf/app.routes (hunk 2 above)
Comment (MJCallahanPage) [inline conf/app.routes]: This has been written to accept a querystring of parameters

**Pull request merged**